torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -2634,6 +2634,13 @@ bool any_variable_defined(variable_list& variables) {
   return false;
 }
 
+at::Tensor nan_to_num_backward(const at::Tensor& grad, const at::Tensor& self) {
+  auto nan_mask = at::isnan(self);
```

Comment:
Instead of isnan and isinf, can you use https://pytorch.org/docs/master/generated/torch.isfinite.html?highlight=isfinite#torch.isfinite ?

Reply:
Ah. Totally! Embarrassed to have missed that!